### PR TITLE
Improve the guide and add a link to the quickstart

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

**Comment by loicmathieu** [comment]
It adds a link to the quickstart that will be provided by this PR: https://github.com/quarkusio/quarkus-quickstarts/pull/627

File: docs/src/main/asciidoc/gcp-functions.adoc [hunk 1]
```diff
@@ -27,9 +27,16 @@ To complete this guide, you need:
 
 == Solution
 
-This guide walks you through generating a sample project followed by creating multiple functions showing how to implements `HttpFunction`, `BackgroundFunction` and `RawBackgroundFunction` in Quarkus..
+This guide walks you through generating a sample project followed by creating multiple functions showing how to implements `HttpFunction`, `BackgroundFunction` and `RawBackgroundFunction` in Quarkus.
```
Comment (gsmet) [inline docs/src/main/asciidoc/gcp-functions.adoc:30]:
```suggestion
This guide walks you through generating a sample project followed by creating multiple functions showing how to implement `HttpFunction`, `BackgroundFunction` and `RawBackgroundFunction` in Quarkus.
```

File: docs/src/main/asciidoc/gcp-functions.adoc [hunk 2]
```diff
@@ -45,6 +52,13 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -Dextensions="google-cloud-functions"
 ----
 
+Now, let's remove what's not needed inside the generated application:
+
+- Remove the dependency `io.quarkus:quarkus-reasteasy` from you `pom.xml` file.
```
Comment (gsmet) [inline docs/src/main/asciidoc/gcp-functions.adoc:57]:
```suggestion
- Remove the dependency `io.quarkus:quarkus-reasteasy` from your `pom.xml` file.
```

File: docs/src/main/asciidoc/gcp-functions.adoc [hunk 3]
```diff
@@ -281,6 +304,13 @@ gcloud beta functions deploy quarkus-example-pubsub \
 The entry point must always be set to `io.quarkus.gcp.functions.QuarkusBackgroundFunction` as this is the class that integrates Cloud Functions with Quarkus.
 ====
 
+To trigger the vent, you can send a file to the `hello_topic` topic of you can use gcloud to simulate one:
```
Comment (gsmet) [inline docs/src/main/asciidoc/gcp-functions.adoc:307]:
```suggestion
To trigger the event, you can send a file to the `hello_topic` topic of you can use gcloud to simulate one:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Spotted a couple of typos.

**Review by gsmet — APPROVED** [review]

**Pull request merged**